### PR TITLE
Text enhancements on tutorial introductions

### PR DESCRIPTION
**Pull request by aliahcodes**
Rather small changes regarding wording, grammar and document structure.

### CODE REVIEW

File: mevislab.github.io/content/examples/data_objects/contours/example1/index.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ In this example, contours are created and colors and styles of these CSOs are cu
 ![Screenshot](/examples/data_objects/contours/example1/image.png)
 
 ## Summary
-+ Contours are stored as their own abstract data type called Contour Segmented Objects. (Often abbreviated to *CSO*.)
++ Contours are stored as their own abstract data type called Contour Segmented Objects (Often abbreviated to *CSO*).
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/examples/data_objects/contours/example1/index.md:38]: Often -> often

File: mevislab.github.io/content/tutorials/openinventor.md [hunk 2]
```diff
@@ -16,34 +16,39 @@ menu:
 # Open Inventor modules {#TutorialOpenInventorModules}
 ## Introduction
 
-Besides blue modules (ML modules) and brown Macro modules, there is a third type of modules, called Open Inventor modules. These modules are green and start with the letters `So\*` (for Scene Objects). Open Inventor modules process and render 3D scene objects and enable image interactions. Scene objects are transmitted via round input and output connectors. With the help of these modules, Open Inventor scenes can be implemented.
+In total, there are three types of modules:
+* Blue ML modules
+* Brown macro modules
+* Green Open Inventor modules
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/openinventor.md:22]: enumerations should start with a lower-case letter, if they are not sentences

File: mevislab.github.io/content/tutorials/openinventor.md [hunk 3]
```diff
@@ -16,34 +16,39 @@ menu:
...
 
-This chapter will start with an example Open Inventor scene. Explanations to Open Inventor scenes will follow the example.
+The names of Open Inventor modules start with the prefix `So\*` (for Scene Objects). Open Inventor modules process and render 3D scene objects and enable image interactions. Scene objects are transmitted using the round input and output connectors. With the help of these modules, Open Inventor scenes can be implemented.
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/openinventor.md:24]: round input and output connectors -> connectors represented as/shown as semicircles?

File: mevislab.github.io/content/tutorials/openinventor.md [hunk 4]
```diff
@@ -16,34 +16,39 @@ menu:
...
 
-{{<alert class="info" caption="Open Inventor Scenes">}}
-The following shows the order in which the modules are executed. The red arrow shows the order of traversal, from top to bottom and left to right. The numbers designate the order in which each module is passed first, from 1 to 8. As shown in the example before, the order of transversal is important.
-
-![Traversing in Open Inventor](/images/tutorials/openinventor/OI1_13.png "Traversing in Open Inventor")
-
+{{<alert class="info" caption="Execution order in Open Inventor scenes">}}
+The model below depicts the order in which the modules are executed. The red arrow indicates the traversal order, from top to bottom and from left to right. The modules are numbered accordingly, from 1 to 8. Knowing about the transversal order can be crucial to achieve a certain ouput.
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/openinventor.md:33]: transversal order -> traversal order

File: mevislab.github.io/content/tutorials/openinventor.md [hunk 5]
```diff
@@ -16,34 +16,39 @@ menu:
...
+![Traversing in Open Inventor](/images/tutorials/openinventor/OI1_13.png "Traversing through a network of Open Inventor modules")
 {{</alert>}}
 
 ## SoGroup and SoSeparator
-The `SoGroup` and `Soseparator` modules cam be used as a container for the child nodes. They both allow multiple inputs and combine the results as one single output as seen above. Nevertheless, there is a big difference in handling the traversal state of the scene graph.
+The `SoGroup` and `Soseparator` modules can be used as containers for child nodes. They both allow multiple inputs and combine the results in one single output as seen above. Nevertheless, there is a big difference in handling the traversal state of the scene graph.
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/openinventor.md:38]: Soseparator -> SoSeparator

File: mevislab.github.io/content/tutorials/thirdparty.md [hunk 6]
```diff
@@ -16,42 +16,43 @@ menu:
 # MeVisLab Tutorial Chapter VIII {#TutorialChapter8}
 
 ## Using ThirdParty software integrated into MeVisLab {#TutorialThirdParty}
-MeVisLab comes with a lot of software already integrated and ready to use. Even if these tools are not available as a module, like itk and vtk for example, they can be used via Python scripting. This chapter shall give some examples of how to use a selection of integrated ThirdParty components.
+MeVisLab is equipped with a lot of useful software right out of the box, like the Insight Segmentation and Registration Toolkit (ITK) or the Visualization Toolkit (VTK). This chapter works as a guide on how to implement some third party component's functionalities to your MeVisLab projects via Python scripting.
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/thirdparty.md:19]: Maybe someting like: `This chapter works as a guide on how to use some of the third party components integrated in MeVisLab for your projects via Python scripting.`

File: mevislab.github.io/content/tutorials/thirdparty.md [hunk 7]
```diff
@@ -16,42 +16,43 @@ menu:
...
-* produce 3D point clouds from stereo cameras,
-* stitch images together to produce a high resolution image of an entire scene,
-* find similar images from an image database,
-* remove red eyes from images taken using flash,
-* follow eye movements,
-* recognize scenery and
-* establish markers to overlay with augmented reality, etc.
+[OpenCV](https://opencv.org/ "OpenCV") (Open Source Computer Vision Library) is an open source computer vision and machine learning software library. 
+OpenCV includes, among others, algorithms to:
+* Detect and recognize faces
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/thirdparty.md:28]: see above for capitalization

File: mevislab.github.io/content/tutorials/thirdparty.md [hunk 8]
```diff
@@ -16,42 +16,43 @@ menu:
...
+* Stitch images together to produce a high resolution image of an entire scene
+* Find similar images from an image database
+* Remove red eyes from images taken using flash
+* Follow eye movements
+* Recognize scenery 
+* Establish markers to overlay with augmented reality
 
 ### assimp
-The [THE ASSET IMPORTER LIBRARY](http://www.assimp.org/) supports loading and processing geometric scenes from various well known 3D formats. MeVisLab uses assimp to import these files and use the scenes directly in MeVisLab.
+The [THE ASSET IMPORTER LIBRARY](http://www.assimp.org/) supports loading and processing geometric scenes from various well known 3D formats. MeVisLab uses assimp to import these files and reuses the scenes directly in MeVisLab.
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/thirdparty.md:43]: "uses" is correct, there is no reuse here

File: mevislab.github.io/content/tutorials/thirdparty.md [hunk 9]
```diff
@@ -16,42 +16,43 @@ menu:
...
+The tutorials available here shall provide examples on how to integrate AI into MeVisLab. You can also integrate other Python AI packages the same way.
 
 ### matplotlib
 [Matplotlib](https://matplotlib.org/) is a library for creating static, animated, and interactive visualizations in Python.
 
-* Create publication quality plots.
-* Make interactive figures that can be zoomed, panned and updated.
-* Customize visual style and layout.
-* Export to many file formats.
+* Create publication quality plots
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/thirdparty.md:55]: same here

File: mevislab.github.io/content/tutorials/visualization.md [hunk 10]
```diff
@@ -14,13 +14,17 @@ menu:
 ---
 # Visualization in MeVisLab {#TutorialVisualization}
 ## Introduction
-MeVisLab contains a whole toolkit to visualize data and images. 2D, 3D and 4D rendering is possible as well as the interaction with images and data objects.
 
-In this chapter, we focus on 2D and 3D visualization and simple image interactions. Not only pixel- and voxel-based data can be visualized, but also scene objects and 3D scenes (see [here](/tutorials/openinventor#TutorialOpenInventorModules)).
+Images and data objects can be rendered in 2D, 3D and 4D and interacted with in several ways using a set of tools available through MeVisLab.
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/visualization.md:18]: I would remove 4D here - you can't really render in 4D...

Comment (aliahcodes) [reply mevislab.github.io/content/tutorials/visualization.md:18]: I just kept it in as it was there before - will be removed.

File: mevislab.github.io/content/tutorials/openinventor.md [hunk 11]
```diff
@@ -16,34 +16,39 @@ menu:
 # Open Inventor modules {#TutorialOpenInventorModules}
 ## Introduction
 
-Besides blue modules (ML modules) and brown Macro modules, there is a third type of modules, called Open Inventor modules. These modules are green and start with the letters `So\*` (for Scene Objects). Open Inventor modules process and render 3D scene objects and enable image interactions. Scene objects are transmitted via round input and output connectors. With the help of these modules, Open Inventor scenes can be implemented.
+In total, there are three types of modules:
+* blue ML modules
+* Brown macro modules
+* Green Open Inventor modules
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/openinventor.md:22]: lower case is for each line

File: mevislab.github.io/content/tutorials/thirdparty.md [hunk 12]
```diff
@@ -16,42 +16,42 @@ menu:
...
+* Track camera movements
+* Track moving objects
+* Extract 3D models of objects
+* Produce 3D point clouds from stereo cameras
+* Stitch images together to produce a high resolution image of an entire scene
+* Find similar images from an image database
+* Remove red eyes from images taken using flash
+* Follow eye movements
+* Recognize scenery 
+* Establish markers to overlay with augmented reality
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/tutorials/thirdparty.md:40]: same here

**Review by mrbean-bremen — APPROVED** [review]

**Pull request merged**